.github/actions/cypress-e2e-testing/action.yml

```diff
@@ -58,7 +58,7 @@ runs:
     - name: Cypress end-to-end testing
       uses: cypress-io/github-action@v6
       with:
-        command: pnpm run cypress:run --spec cypress/e2e/**/${{ inputs.feature_file_path }} 
+        command: pnpm run cypress:run --spec integration/**/${{ inputs.feature_file_path }} 
```

Comment:
This is supposed to be e2e so integration is not really accurate

Reply:
the tree is now 
tests -> e2e -> integration 
if i rename it to e2e it will be 
tests -> e2e -> e2e

Reply:
then move `tests/e2e/cypress/e2e` to `tests/e2e/features`